examples/map/difference_images.py

```diff
@@ -0,0 +1,62 @@
+"""
+===========================
+Plotting a difference image
+===========================
+How to compute and plot a difference image.
+The example uses `sunpy.map.MapSequence` to compute a difference image and then plot it.
+This basic method works for base difference or running difference. Just change whether
+you're subtracting the previous image or the first image in a sequence.
+"""
+###########################################################################
+
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import astropy.units as u
+from sunpy.net import Fido, attrs as a
+import sunpy.map
+
+###########################################################################
+# First we'll download a couple images and store them in a
+# `sunpy.map.MapSequence`. This could be from any channel of any imager.
+# Here, we use SDO/AIA 304 Å.
+
+instrument = a.Instrument('AIA')
+wave = a.Wavelength(30 * u.nm, 31 * u.nm)
+result = Fido.search(a.Time('2015-06-18T00:00:00', '2015-06-18T00:00:10') |
+                     a.Time('2015-06-18T01:03:30', '2015-06-18T01:03:35'), instrument, wave)
+downloaded_files = Fido.fetch(result)
+maps = sunpy.map.Map(downloaded_files, sequence=True)
+
+###########################################################################
+# Now we'll do a standard plot of the second image just to see it.
+
+plt.figure()
+ax = plt.subplot(projection=maps[1])
+maps[1].plot()
+
+###########################################################################
+# And now we can do take the actual difference.
+
+diff = maps[1].data - maps[0].data
+
+###########################################################################
+# But we have to decide what to do with the metadata. For example, what
+# time does this difference image correspond to? The time of the first or
+# second image? The mean time? You'll have to decide what makes most sense
+# for your application. Here we'll just use the metadata from the second
+# image. Then we can store the difference and header back in a Map.
+
+meta = maps[1].meta
+diff_map = sunpy.map.Map(diff, meta)
+
+###########################################################################
+# Finally, we'll plot it. We'll apply a colormap and renormalize the
+# intensity so that it shows up well.
+
+plt.figure()
+ax_diff = plt.subplot(projection=diff_map)
+diff_map.plot(cmap='Greys_r',
+              norm=colors.Normalize(vmin=-50, vmax=50))
+plt.show()
+
+#sphinx_gallery_thumbnail_number = 2
```

Comment:
We should do this everywhere.

Reply:
Why is this showing up at all?!

Reply:
I searched through the sphinx-gallery and could not find an example of where this is used. Is the expectation that it just shows up in the example code?

Reply:
Aren't all comments rendered by sphinx? Why would this one be any different.

Reply:
Looks like my sphinx n00bness commented on in the initial PR is turning out to be the sticking point. I have no idea how that stuff works, I just saw that setting the thumbnail ≠ 1 was an option in the template (examples/example_template/example_template.py). Guess we've uncovered something unexpected.